src/app/app-modules/core/components/beneficiary-worklist/beneficiary-worklist.component.html

```
@@ -0,0 +1,156 @@
+<div [class]="padded ? 'px-6 pb-10 pt-3' : 'pb-10'">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)